index.html

```diff
@@ -89,89 +94,153 @@
 <script>
   var data = [];
   var options = {
-    yearRange: 'all'
+    location: 'all'
   };
-
   var svg, x, y;
-
-  var margin = { top: 15, right: 15, bottom: 30, left: 45 };
-
-  var width = 640 - margin.right - margin.left;
-  var height = 400 - margin.top - margin.bottom;
-
+  var margin = { top: 15, right: 15, bottom: 30, left: 90 };
+  var width = 900 - margin.right - margin.left;
+  var height = 500 - margin.top - margin.bottom;
+  var binCount = 54
+  var binCounty = 15
   // FETCH DATA
-
   d3.json('data.json', function (error, json) {
     if (error) { throw error; }
-
     data = json;
-
-    setup();
-    update();
+    var chart = new Chart();
   });
+      function Chart() {
+        var chart = this;
 
-  function setup() {
-    // SVG
-
-    svg = d3.select('#chart')
+        chart.svg = d3.select('#chart1')
       .append('svg')
       .attr('width', width + margin.right + margin.left)
       .attr('height', height + margin.top + margin.bottom)
       .append('g')
       .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
-
     // SCALES
-
-    x = d3.scale.linear()
-      .domain([0, d3.max(data, function (d) { return d.unemployment; })])
+    chart.x = d3.scale.linear()
+      .domain(d3.extent(data, function (a) {return a.year; }))
       .range([0, width])
       .nice();
-
-    y = d3.scale.linear()
-      .domain(d3.extent(data, function (d) { return d.inflation; }))
+    chart.y = d3.scale.linear()
+      .domain(d3.extent(data, function (a) {return a.value; }))
       .range([height, 0])
       .nice();
-
+    
     // AXES
-
-    // EVENT HANDLERS
-
-    d3.select('#year-range').on('change', function () {
-      options.yearRange = d3.event.target.value;
+    // format x axis
+    var formatyears = d3.format("1000")
+    // format y axis
+    var formatmoney = d3.format("$1,000,000,000")
+    //Define X axis
+   
+    var xAxis = d3.svg.axis()
+             .scale(x)
+             .orient("bottom")
+             .ticks(20)
+             .tickSize(height + 15)
+             .tickFormat(formatyears);
+    //Define Y axis
+    var yAxis = d3.svg.axis()
+             .scale(y)
+             .tickSize(width)
+             .tickFormat(formatmoney)
+             .orient("right")
+             .ticks(5);
+   
+    //Create X axis
+    var chart.gx = svg.append("g")
+          .attr("class", "axis")
```

Comment:
So this is a really understandable mistake, but when you set a property of a constructor function, you don't use 'var'. So above, you just want:

```js
chart.gx = svg.append("g")
```